### PR TITLE
Refactoring scr.prompt and scr.wheel

### PR DESCRIPTION
**Pull request by faridtsl**
Refactoring scr.prompt and scr.wheel

### CODE REVIEW

File: libr/core/cconfig.c [hunk 1]
```diff
@@ -2765,10 +2765,10 @@ R_API int r_core_config_init(RCore *core) {
 #else
 	SETPREF ("scr.responsive", "false", "Auto-adjust Visual depending on screen (e.g. unset asm.bytes)");
 #endif
-	SETPREF ("scr.wheelnkey", "false", "Use sn/sp and scr.nkey on wheel instead of scroll");
+	SETPREF ("scr.wheel.nkey", "false", "Use sn/sp and scr.nkey on wheel instead of scroll");
 	SETPREF ("scr.wheel", "true", "Mouse wheel in Visual; temporaryly disable/reenable by right click/Enter)");
 	SETPREF ("scr.atport", "false", "V@ starts a background http server and spawns an r2 -C");
```
Comment (XVilka) [inline libr/core/cconfig.c]: move outside of the "wheel" section maybe?

Comment (faridtsl) [reply libr/core/cconfig.c]: scr.nkey is already taken and has possible values of [fun, hit, flag].

Comment (radare) [reply libr/core/cconfig.c]: scr.wheel.nkey lgtm

**Comment by radare** [comment]
this pr contains the changes in asm.cmt* right? pls update the testsuite or we get >35 XX

**Comment by faridtsl** [comment]
No asm.cmt is in the other PR. sorry !

**Pull request merged**